webpack/webpack.config.js

```diff
@@ -34,6 +34,11 @@ const externals = {
 module.exports = function( env = { environment: "production", recalibration: "disabled" } ) {
 	const mode = env.environment || process.env.NODE_ENV || "production";
 	const isRecalibration = ( env.recalibration || process.env.YOAST_RECALIBRATION || "disabled" ) === "enabled";
+	const allowedHosts = [
+		"local.wordpress.test",
+		"build.wordpress-develop.test",
+		"src.wordpress-develop.test",
+	].concat( ( process.env.ALLOWED_HOSTS || "" ).split( " " ) );
```

Comment:
Seems to me if `process.env.ALLOWED_HOSTS` is undefined, then the returned array will have a 4th empty item e.g. `["local.wordpress.test", "build.wordpress-develop.test", "src.wordpress-develop.test", ""]`.
Not sure if this might be a problem, what would mean for webpack that a `""` host is allowed?

Reply:
Yes, I do not know and would be difficult to find out. Instead, I refactored it so that does not happen anymore.
In the process, I included more comments and moved the defaults to a variable outside of the function.